### PR TITLE
Fix Travis build failures introduced by memory region packet changes

### PR DESCRIPTION
**Pull request by fjricci**
lol@lldbtestsuite

### CODE REVIEW

File: Support/Testing/Patches/memory-region-info.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff --git a/packages/Python/lldbsuite/test/tools/lldb-server/gdbremote_testcase.py b/packages/Python/lldbsuite/test/tools/lldb-server/gdbremote_testcase.py
+index d63ddbe..68f4f50 100644
+--- a/packages/Python/lldbsuite/test/tools/lldb-server/gdbremote_testcase.py
++++ b/packages/Python/lldbsuite/test/tools/lldb-server/gdbremote_testcase.py
+@@ -698,7 +698,7 @@ class GdbRemoteTestCaseBase(TestBase):
+ 
+         # Validate keys are known.
+         for (key, val) in list(mem_region_dict.items()):
+-            self.assertTrue(key in ["start", "size", "permissions", "error"])
++            self.assertTrue(key in ["start", "size", "permissions", "error", "name"])
```
Comment (sas) [inline Support/Testing/Patches/memory-region-info.patch]: This is funny. This was added because upstream needs it now. I think when we update to 4.0 test suite, you'll have to revert this change.

Comment (fjricci) [inline Support/Testing/Patches/memory-region-info.patch]: Well I mean, presumably since upstream needs it, they'll want this patch. Actually how is upstream passing tests without this...

Comment (fjricci) [inline Support/Testing/Patches/memory-region-info.patch]: Oh wait upstream added this. I'll cherry-pick their patch instead.

Comment (fjricci) [inline Support/Testing/Patches/memory-region-info.patch]: nvm, I can't do that, it's too big of a patch. I'll just keep this around until we update and get a merge conflict.
